app/src/main/java/com/yello/presentation/onboarding/Activity/OnBoardingActivity.kt

```diff
@@ -0,0 +1,51 @@
+package com.yello.presentation.onboarding.Activity
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.widget.ViewPager2
+import com.example.ui.base.BindingActivity
+import com.yello.R
+import com.yello.databinding.ActivityOnboardingBinding
+import com.yello.presentation.onboarding.fragment.AddFriendFragment
+import com.yello.presentation.onboarding.fragment.CodeFragment
+import com.yello.presentation.onboarding.fragment.GenderFragment
+import com.yello.presentation.onboarding.fragment.NameIdFragment
+import com.yello.presentation.onboarding.fragment.SchoolFragment
+import com.yello.presentation.onboarding.fragment.StartAppFragment
+import com.yello.presentation.onboarding.fragment.StudentIdFragment
+import com.yello.presentation.onboarding.ViewPagerFragmentAdapter
+
+class OnBoardingActivity :
+    BindingActivity<ActivityOnboardingBinding>(R.layout.activity_onboarding) {
+    private val data = mutableListOf<String>()
+    private val fragmentList = ArrayList<Fragment>()
+    private lateinit var viewPager: ViewPager2
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityOnboardingBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        castView()
+        addToList()
+        fragmentList.add(SchoolFragment())
+        fragmentList.add(StudentIdFragment())
+        fragmentList.add(NameIdFragment())
+        fragmentList.add(GenderFragment())
+        fragmentList.add(AddFriendFragment())
+        fragmentList.add(CodeFragment())
+        fragmentList.add(StartAppFragment())
+
+        viewPager.adapter = ViewPagerFragmentAdapter(this, fragmentList)
+        viewPager.orientation = ViewPager2.ORIENTATION_HORIZONTAL
+    }
+
+    private fun castView() {
+        viewPager = findViewById(R.id.view_pager2)
```

Comment:
```suggestion
        viewPager = binding.viewPager2
```
viewBinding을 활용하여 뷰를 참조하는 것이 더 코틀린스럽고 빠르기도 하고 오류 발생 확률이 적어 좋을 것 같습니다!